imagetool/src/main/java/com/oracle/weblogic/imagetool/aru/AruProduct.java

```diff
@@ -25,7 +25,8 @@
     WCS("20995", "Oracle WebCenter Sites"),
     JDEV("11281", "Oracle JDeveloper"),
     OPSS("16606", "Oracle Platform Security Service"),
-    OWSM("12787", "Oracle Webservices Manager")
+    OWSM("12787", "Oracle Webservices Manager"),
```

Comment:
This should be `Oracle Web Services Manager`

Reply:
Unfortunately, no.  This text must match the text used in ARU releases.